code/Web Api Final Project/home.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>HTML and CSS Final Project</title>
+    <link rel="stylesheet" href="homestyle.css" />
+</head>
+<body>
+    <div class="landing-page">
+        <div class="header">
+            <h1><em>What's new?</em></h1>
+            <button>New Post</button>
+            <br>
+        </div>
+        <hr>
+        <div class="topics">
+            <div class="trending">
+                <h2>Trending</h2>
+            </div>
+            <div class="communities">
+                <h2>Communities</h2>
+            </div>
+            <div class="profile">
+                <h2>Profile</h2>
+            </div>
+        </div>
+    <div class="posts">
+        <div class="user-posts">

```

Comment:
It's better to just have empty elements than to have dummy data that never actually loads. I would take out all of the material that ultimately gets overwritten